### PR TITLE
Implemented Cephalid Snitch (Torment)

### PR DESCRIPTION
**Pull request by tcontis**

### CODE REVIEW

File: Mage.Sets/src/mage/cards/c/CephalidSnitch.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+public class CephalidSnitch extends CardImpl {
+
+    public CephalidSnitch(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId,setInfo,new CardType[]{CardType.CREATURE},"{1}{U}");
+        this.subtype.add(SubType.CEPHALID);
+        this.subtype.add(SubType.WIZARD);
+        this.power = new MageInt(1);
+        this.toughness = new MageInt(1);
+
+        // Sacrifice Cephalid Snitch: Target creature loses protection from black until end of turn.
```
Comment (LevelX2) [inline Mage.Sets/src/mage/cards/c/CephalidSnitch.java]: I don't think this will work for example to remove half of the protection from Akroma's Memorial.

Comment (tcontis) [reply Mage.Sets/src/mage/cards/c/CephalidSnitch.java]: Tested it and it does indeed remove all protection. I'll have to probably make a new effect.

**Comment by tcontis** [comment]
I'm trying to write some updated code which modifies the protection abilities. Everything works except getProtectionAbilities(). For Auriok Champion and Animar, red, white and black show up. I'm a bit confused.

**Comment by tcontis** [comment]
Well everything's screwed up now with the commits. Always happens. :/

**Pull request closed without merging**